install_config/master_node_configuration.adoc

```diff
@@ -1199,16 +1199,17 @@ The advanced audit feature provides several improvements over the
 xref:master-node-config-audit-config[basic audit functionality], including
 fine-grained events filtering and multiple output back ends.
 
-To enable the advanced audit feature, provide the following values in the
-`openshift_master_audit_config` parameter:
+To enable the advanced audit feature using a policy file, provide the following values in the
+`openshift_master_audit_config` and `openshift_master_audit_policyfile` parameters:
 
 ----
 openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/lib/origin/audit-ocp.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/origin/master/adv-audit.yaml", "logFormat":"json"}
+openshift_master_audit_policyfile="/path/to/adv-audit.yaml"
 ----
 
 [IMPORTANT]
 ====
-The policy file *_/etc/origin/master/adv-audit.yaml_* must be available on each master node.
+The policy file *_/path/to/adv-audit.yaml_* must be available before installation.
```

Comment:
I'd say something like "You must create the `adv-audit.yaml` file before you install the cluster and specify its location in the cluster inventory file."

Reply:
Your correction is easier to know parameter usage. Thanks.